test/Scripts.Tests/package-release.zip.snapshot

```diff
@@ -1,323 +1,61 @@
-Plugins/iOS/Device/Sentry.framework/Headers/SentrySpanContext.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentrySpanId.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentrySpanId.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentrySpanProtocol.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentrySpanProtocol.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentrySpanStatus.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentrySpanStatus.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentryStacktrace.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentryStacktrace.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentryThread.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentryThread.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentryTraceHeader.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentryTraceHeader.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentryTransactionContext.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentryTransactionContext.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentryUser.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentryUser.h.meta
-Plugins/iOS/Device/Sentry.framework/Headers/SentryUserFeedback.h
-Plugins/iOS/Device/Sentry.framework/Headers/SentryUserFeedback.h.meta
-Plugins/iOS/Device/Sentry.framework/Modules/module.modulemap
-Plugins/iOS/Device/Sentry.framework/Modules/module.modulemap.meta
-Plugins/Android/Sentry.meta
+Plugins/Android/
 Plugins/Android/Sentry/gson-2.8.5.jar
-Plugins/Android/Sentry/gson-2.8.5.jar.meta
 Plugins/Android/Sentry/sentry-5.5.0.jar
-Plugins/Android/Sentry/sentry-5.5.0.jar.meta
 Plugins/Android/Sentry/sentry-android-core-release.aar
-Plugins/Android/Sentry/sentry-android-core-release.aar.meta
 Plugins/Android/Sentry/sentry-android-ndk-release.aar
-Plugins/Android/Sentry/sentry-android-ndk-release.aar.meta
+Plugins/iOS/SentryNativeBridge.m
+Plugins/iOS/Device/Sentry.framework.meta
+Plugins/iOS/Simulator/Sentry.framework.meta
 Runtime/io.sentry.unity.runtime.asmdef
 Runtime/io.sentry.unity.runtime.asmdef.meta
+Runtime/Microsoft.Bcl.AsyncInterfaces.dll
 Runtime/Sentry.dll
 Runtime/Sentry.dll.meta
-Runtime/Sentry.Microsoft.Bcl.AsyncInterfaces.dll
-Runtime/Sentry.Microsoft.Bcl.AsyncInterfaces.dll.meta
-Runtime/Sentry.System.Buffers.dll
-Runtime/Sentry.System.Buffers.dll.meta
-Runtime/Sentry.System.Collections.Immutable.dll
-Runtime/Sentry.System.Collections.Immutable.dll.meta
-Runtime/Sentry.System.Memory.dll
-Runtime/Sentry.System.Memory.dll.meta
-Runtime/Sentry.System.Numerics.Vectors.dll
-Runtime/Sentry.System.Numerics.Vectors.dll.meta
-Runtime/Sentry.System.Reflection.Metadata.dll
-Runtime/Sentry.System.Reflection.Metadata.dll.meta
-Runtime/Sentry.System.Runtime.CompilerServices.Unsafe.dll
-Runtime/Sentry.System.Runtime.CompilerServices.Unsafe.dll.meta
-Runtime/Sentry.System.Text.Encodings.Web.dll
-Runtime/Sentry.System.Text.Encodings.Web.dll.meta
-Runtime/Sentry.System.Text.Json.dll
```

Comment:
This is removing all the aliased assemblies and for sure won't match the final package